Proposed/asc056.md

```diff
@@ -0,0 +1,53 @@
+#### **Title**
+Add `informatics` to Autoprotocol instruction schema
+
+#### **Authorship**
+Asuka Ota <asuka.ota@strateos.com>
+
+#### **Motivation**
+In the current form, Autoprotocol is unable to capture a scientific intent at the molecular
+level upon execution of an `instruction`. One of the examples is pcr whereby a mixture of nucleotides
+and polymerase are mixed in a vial to synthesize new molecules via `thermocycle`. In another case, a user may want
+to purify a compound of interest from a mixture of compounds in instructions such as `spe` and/or `lcms`. In 
+addition to the physical execution parameters, there should be a set of parameters describing the intended effects 
+of an instruction on the sample composition.
+
+#### **Proposal**
+Instruction currently has `op`, and `data` attributes. We propose adding a new attribute called `informatics` 
+in our Autoprotocol schema to represent the intended effects of an instruction. A vendor may choose to integrate 
+the data specified in `informatics` with their internal LIMS systems. This `informatics` is an optional parameter 
+that can accept a list of effects for each instruction. `type` specifies the type of the effect, and `details` is a 
+set of parameters to describe the effect.   
+
+We are proposing to only add 'attach_compounds' at this time, but more types 
+may be available in the future such as 'purify_compounds', 'liquefy', etc. Here, `refs` is a list of aliquots that are 
+affected by the instruction. The instruction must directly operate on `refs` or its container to have any effects. 
```

Comment:
`refs` typically refers to containers in autoprotocol, not wells. Would it be possible to rename to `wells`?